cmd/geth/accountcmd_test.go

```diff
@@ -43,13 +43,13 @@ func tmpDatadirWithKeystore(t *testing.T) string {
 }
 
 func TestAccountListEmpty(t *testing.T) {
-	geth := runGeth(t, "account", "list")
+	geth := runGeth(t, "--nousb", "account", "list")
```

Comment:
I might remove the `--nousb` if not really needed, as in the next go-ethereum merge, the revert the flag to `--usb`

Reply:
Done with https://github.com/etclabscore/core-geth/pull/347/commits/c5fe6a635aab6df080197113dd078dfd7eec9603